charts/araali-quickstartnanny/templates/quickstart_nanny.yaml

```diff
@@ -0,0 +1,76 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    is_quickstart: "true"
+  name: araali-quickstartnanny
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: araali-quickstartnanny
+  namespace: araali-quickstartnanny
+  labels:
+    is_quickstart: "true"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: araali-quickstartnanny
+  labels:
+    is_quickstart: "true"
+rules:
+- apiGroups: ["*"]
+  resources: ["*"]
+  verbs: ["*"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: araali-quickstartnanny
+  labels:
+    is_quickstart: "true"
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: araali-quickstartnanny
+roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: araali-quickstartnanny
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+    name: araali-quickstartnanny
+    namespace: araali-quickstartnanny
+    labels:
+        name: araali-quickstartnanny
+spec:
+    template:
+        metadata:
+            labels:
+                is_quickstart: "true"
+                name: araali-quickstartnanny
+        spec:
+            restartPolicy: Never
+            containers:
+            - name: araali-quickstartnanny
+              {{- if not .Values.image }}
```

Comment:
Just have `:prod` tag in the default values.yaml file for the `image` key. You don't need to complicate with this if else logic here.